test/wasm/math.js

```diff
@@ -0,0 +1,12 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+let ffi = {};
+let wasm = readbuffer("math.wasm");
```

Comment:
@Cellule added this as closer analog to how browser behaves (and it is emit by emscripten)